fleetspeak/src/server/comms.go

```diff
@@ -49,13 +49,22 @@ type commsContext struct {
 // GetClientInfo loads basic information about a client. Returns nil if the client does
 // not exist in the datastore.
 func (c commsContext) GetClientInfo(ctx context.Context, id common.ClientID) (*comms.ClientInfo, error) {
-	cld, err := c.s.dataStore.GetClientData(ctx, id)
-	if err != nil {
-		if c.s.dataStore.IsNotFound(err) {
-			return nil, nil
+	var cld *db.ClientData
+	var err error
+	var ch bool
```

Comment:
Does "ch" stand for "cached"? It took me a minute and a look at the return at the end of this function to figure it out. Just naming this "cached" would be much better.

Reply:
changed to cacheHit